src/pages/explore/PayoutsPage/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { ApiPromise } from '@polkadot/api'
+import { AccountId32 } from '@polkadot/types/interfaces'
+import { PalletSocietyPayoutRecord } from '@polkadot/types/lookup'
+import { useEffect, useState } from 'react'
+import { PayoutsList } from './components/PayoutsList'
+import { useAccount } from '../../../account/AccountContext'
+import { useConsts } from '../../../hooks/useConsts'
+import { LoadingSpinner } from '../components/LoadingSpinner'
+import { buildSocietyMembersArray, deriveMembersInfo } from '../helpers'
+
+type PayoutsPageProps = {
+  api: ApiPromise | null
+}
+
+const PayoutsPage = ({ api }: PayoutsPageProps): JSX.Element => {
+  const { activeAccount } = useAccount()
+  const [members, setMembers] = useState<ExtendedSocietyMember[] | null>(null)
+  const [trigger, setTrigger] = useState(false)
+  const society = api?.derive.society
+
+  const { graceStrikes } = useConsts()
+
+  const handleUpdate = () => {
+    setTrigger((prev) => !prev)
+  }
+
+  let defender: AccountId32
+  let skeptic: AccountId32
+
+  useEffect(() => {
+    const fetchData = async () => {
+      if (!api || !society) return
+
+      const info: ExtendedDeriveSociety = await society.info()
+      const defending = await api.query.society.defending()
+
+      defender = defending.unwrap()[0]
+      skeptic = defending.unwrap()[1]
+
+      info.defender = defender
+      info.skeptic = skeptic
+
+      const responseMembers: ExtendedDeriveSociety[] = await deriveMembersInfo(api)
+      const membersArray = buildSocietyMembersArray(responseMembers, info, graceStrikes)
```

Comment:
I'm looking at these responses/types, question:
- What's the definition of member?
- What's the definition of extended member?

(If these are just .ts names and not concepts, please ignore)

---


- How do I know if someone has a payout to claim? 
  - `member.hasPayouts` is enough?
  - Should I look into SocietyMember.payouts?

Reply:
> What's the definition of extended member?

Just an application-specific type, don't worry, it's just a regular member but with extra data.

> member.hasPayouts is enough?

Yep, should be enough.

Reply:
Should I use this method to claim the payout?

https://polkadot.js.org/docs/substrate/extrinsics#claimpayout

Reply:
> Should I use this method to claim the payout?
> 
> https://polkadot.js.org/docs/substrate/extrinsics#claimpayout

No, this is the one: https://github.com/paritytech/polkadot-sdk/blob/924089ff33e35739a21fa57f294297b01e617c00/substrate/frame/society/src/lib.rs#L1009

It's `society.payout()`